### PR TITLE
feat: Include attestation runs in TUI by default

### PR DESCRIPTION
**Pull request by wbxyz**

### CODE REVIEW

File: tools/ctl/ide/ui.go [hunk 1]
```diff
@@ -416,7 +424,7 @@ func (e *explorer) LoadTree() error {
 	}
 	byBench := make(map[string][]string)
 	for _, run := range runs {
-		if run.Type == benchmark.AttestMode {
+		if !e.settings.includeAttest && run.Type == benchmark.AttestMode {
```
Comment (msuozzo) [inline tools/ctl/ide/ui.go:427]: do we even need this constraint? would an alternative be some indication in the UI?

**Review by msuozzo — COMMENTED** [review]
I could see just removing the constraint for now and then potentially re-introducing some visible indicator (possibly along with size / benchmark) to the UI

File: tools/ctl/ide/ui.go (hunk 1 above)
Comment (wbxyz) [reply tools/ctl/ide/ui.go]: I think eventually we will want some setting/filters on the explorer with a way to adjust them in the UI (I'm thinking like filter based on error message, or grouping based on log regex) but I can leave the settings construct out of this PR and just add an indication. The indication is useful regardless.

File: tools/ctl/ide/ui.go [hunk 2]
```diff
@@ -368,7 +369,15 @@ func (e *explorer) makeVerdictGroupNode(vg *rundex.VerdictGroup, percent float32
 }
 
 func (e *explorer) makeRunNode(runid string) *tview.TreeNode {
-	node := tview.NewTreeNode(runid).SetColor(tcell.ColorGreen).SetSelectable(true)
+	var title string
+	if run, ok := e.runs[runid]; ok && run.Type == benchmark.AttestMode {
+		title = fmt.Sprintf("(a)%s", runid)
```
Comment (msuozzo) [inline tools/ctl/ide/ui.go:374]: ugh I don't love the single-letters, especially for new users, but I guess it's fine for now...

would mainly want to keep them the same length and I can't think of two descriptors that are consistent with our current terminology and maintain that property.

File: tools/ctl/ide/ui.go [hunk 3]
```diff
@@ -410,16 +419,16 @@ func (e *explorer) makeRunGroupNode(benchName string, runs []string) *tview.Tree
 func (e *explorer) LoadTree() error {
 	e.root.ClearChildren()
 	log.Printf("Fetching runs...")
+	var err error
```
Comment (msuozzo) [inline tools/ctl/ide/ui.go:422]: is this still necessary?

**Review by msuozzo — APPROVED** [review]

File: tools/ctl/ide/ui.go (hunk 2 above)
Comment (wbxyz) [reply tools/ctl/ide/ui.go:374]: Discussed offline, we decided to move the descriptors to the end of the runid and use longer, more descriptive terms.

File: tools/ctl/ide/ui.go (hunk 3 above)
Comment (wbxyz) [reply tools/ctl/ide/ui.go:422]: Nope, removed.

**Pull request merged**